app/pages/_app.js

```diff
@@ -16,12 +17,23 @@ import 'Styles/theme/simple-line-icons/css/simple-line-icons.css';
 const PageLayout = ({ children }) => children;
 
 class App extends NextApp {
+  state = {
+    hasError: false,
+    error: null,
+  };
+
+  handleError(error) {
```

Comment:
no need to extract `handleError`, won't be used anywhere else.

Reply:
Done